Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.8/dist-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.8/dist-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
clf = BaggingClassifier(base_estimator=DecisionTreeClassifier(),n_estimators=10, random_state=0).fit(X_trainn_min_max, y_trainn_min_max)
rsb = clf.predict(X_testn_min_max)
b = ['Decision Tree']
Tree = pd.DataFrame(rsb,columns = b)
------------------

[0;31m[0m
[0;31mNameError[0mTraceback (most recent call last)
[0;32m<ipython-input-2-e315df66bed9>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mclf[0m [0;34m=[0m [0mBaggingClassifier[0m[0;34m([0m[0mbase_estimator[0m[0;34m=[0m[0mDecisionTreeClassifier[0m[0;34m([0m[0;34m)[0m[0;34m,[0m[0mn_estimators[0m[0;34m=[0m[0;36m10[0m[0;34m,[0m [0mrandom_state[0m[0;34m=[0m[0;36m0[0m[0;34m)[0m[0;34m.[0m[0mfit[0m[0;34m([0m[0mX_trainn_min_max[0m[0;34m,[0m [0my_trainn_min_max[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0mrsb[0m [0;34m=[0m [0mclf[0m[0;34m.[0m[0mpredict[0m[0;34m([0m[0mX_testn_min_max[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0mb[0m [0;34m=[0m [0;34m[[0m[0;34m'Decision Tree'[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mTree[0m [0;34m=[0m [0mpd[0m[0;34m.[0m[0mDataFrame[0m[0;34m([0m[0mrsb[0m[0;34m,[0m[0mcolumns[0m [0;34m=[0m [0mb[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'X_trainn_min_max' is not defined
NameError: name 'X_trainn_min_max' is not defined

